internal/controller/constants.go

```diff
@@ -21,11 +21,12 @@ const (
 	AppLabel = "app"
 
 	// Access strategy label keys
-	LabelWorkspaceName           = "workspace.jupyter.org/workspaceName"
-	LabelWorkspaceNamespace      = "workspace.jupyter.org/workspaceNamespace"
-	LabelAccessStrategyName      = "workspace.jupyter.org/accessStrategyName"
-	LabelAccessStrategyNamespace = "workspace.jupyter.org/accessStrategyNamespace"
-	LabelWorkspaceTemplate       = "workspace.jupyter.org/template"
+	LabelWorkspaceName              = "workspace.jupyter.org/workspaceName"
+	LabelWorkspaceNamespace         = "workspace.jupyter.org/workspaceNamespace"
+	LabelAccessStrategyName         = "workspace.jupyter.org/accessStrategyName"
+	LabelAccessStrategyNamespace    = "workspace.jupyter.org/accessStrategyNamespace"
+	LabelWorkspaceTemplate          = "workspace.jupyter.org/template"
```

Comment:
For consistency, could you rename `LabelWorkspaceTemplateName` and `/template-name` please?

While we are at it, shall we change all label names casing (e.g.` /workspaceName` > `workspace-name`).
Fine as well if we think it's better in a future PR